main.js

```diff
@@ -76,7 +76,46 @@ const printToDom = (divId, textToPrint) => {
 };
 
 // *** HTML Builder Functions *** //
+// Get Package Form Info
+// const packageForm = (e) => {
+//   e.preventDefault();
 
+//   const packName = document.querySelector('#package-form').value;
+//   const packDescription = document.querySelector('#option-package').value;
+  
+// const obj = {
+//   packName,
+//   packDescription,
+// };
+
+// users.push(obj);
+// // createPackage(users.packages);
+
+//   document.querySelector('#add-package').addEventListener('click', packageForm);
+//   document.querySelector('form-control').reset();
+// };
+// End Get Package Form Info
+
+// Create Package Card
+const createPackage = (taco) => {
+  let domString = "";
+  taco.forEach((element) => {
+      element.packages.forEach((arg) => {
+      domString += `<div class="card border-dark mb-3" style="width: 25rem">
+                       <div class="card-header">Package Name</div>
+                       <div class="text">${arg.packageName}</div>
+                       <div class="card-body">
+                       <p class="card-text">Package Description</p>
+                       <div class="text">${arg.packageDescription}</div>
```

Comment:
Put your arg.packagName on line 105 and put your arg.packadageDescription on line 108